definitions/npm/redux-actions_v2.x.x/flow_v0.39.x-/redux-actions_v2.x.x.js

```diff
@@ -58,6 +58,7 @@ declare module "redux-actions" {
   declare function createActions(...identityActions: string[]): Object;
 
   declare type Reducer<S, A> = (state: S, action: A) => S;
+  declare type ReduxReducer<S, A> = (state: S | void, action: A) => S;
```

Comment:
Can you add some commentary on the difference between these two? It's not obvious, and would be helpful to others.

Reply:
@AndrewSouthpaw I've added some commentary. Hopefully it's sufficient.